.github/workflows/rpc-parity.yml

```diff
@@ -0,0 +1,38 @@
+name: RPC parity tests
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+jobs:
+  rpc-parity:
+    name: RPC parity tests
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v4
+      - name: Run api compare tests on calibnet
+        shell: bash
+        run: |
```

Comment:
Any reason to delete the `Remove unnecessary software`? Given the increasing snapshot sizes, it was important.

Reply:
Added back.